### PR TITLE
Replace Java8 syntax with Java7

### PR DESCRIPTION
**Pull request by bivas**
Since Gradle 3.x requires support for Java7 I've updated the plugin to use only Java7 syntax

Related to Issue #26

### CODE REVIEW

**Comment by bivas** [comment]
I've tried running this plugin with environment which has JDK7 and the groovy files caused build failures with `java.util.function.Consumer` not found (as it only exists in JDK8).

Groovy still needs to compile to Java. And the syntax I removed is not Groovy, but Java specific e.g. look at the replacement of `forEach` with `each` (which is true Groovy syntax).

**Comment by bivas** [comment]
@zvezdan If you like, I can use Groovy's `TimeDuration` instead of Java8 Duration.

File: pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy [hunk 1]
```diff
@@ -164,7 +163,7 @@ class BuildWheelsTask extends DefaultTask {
                 if (pythonExtension.consoleOutput == ConsoleOutput.RAW) {
                     LOGGER.lifecycle(stream.toString().trim())
                 } else {
-                    String prefix = String.format(messageHead + ' (%d:%02d.%03d s)', duration.toMinutes(), duration.getSeconds() % 60, (int)(duration.getNano() / 1000000))
+                    String prefix = String.format("Install (%d s)", TimeUnit.MILLISECONDS.toSeconds(duration))
```
Comment (zvezdan) [inline pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy]: Why was `messageHead` changed to `"Install"` here?
The message was different in this task compared to the other file below.

**Comment by zvezdan** [comment]
@bivas I'll let @ethankhall decide on the preference between Groovy/Java features used.

File: pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy [hunk 2]
```diff
@@ -164,7 +162,7 @@ class BuildWheelsTask extends DefaultTask {
                 if (pythonExtension.consoleOutput == ConsoleOutput.RAW) {
                     LOGGER.lifecycle(stream.toString().trim())
                 } else {
-                    String prefix = String.format(messageHead + ' (%d:%02d.%03d s)', duration.toMinutes(), duration.getSeconds() % 60, (int)(duration.getNano() / 1000000))
+                    String prefix = String.format(messageHead + " (%d:%02d s)", duration.minutes, duration.seconds % 60)
```
Comment (zvezdan) [inline pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy]: Are we losing milliseconds now?

Comment (zvezdan) [inline pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy]: Are we losing milliseconds now?
Also on the similar line in the other file below.

**Comment by zvezdan** [comment]
@ethankhall Can you also take a look before the merge?

**Comment by ethankhall** [comment]
Sorry I was traveling. Looks good, merging.

**Pull request merged**